### PR TITLE
Php

### PR DESCRIPTION
**Pull request by jeronimo13**
Note: branch out from https://github.com/airdeploy/airdeploy-docs/pull/19

### CODE REVIEW

File: docs/api-reference/go.md [hunk 1]
```diff
@@ -84,7 +84,7 @@ flagger.Shutdown(5 * time.Second)
 func (flagger *Flagger) Publish(entity *core.Entity)
 ```
 
-Explicitly notify Airdeploy about an Entity
+Explicitly notifies Airdeploy about an Entity
```
Comment (thealveen) [inline docs/api-reference/go.md:87]: this grammar correction is incorrect. revert this

File: docs/api-reference/js.md [hunk 2]
```diff
@@ -90,7 +90,7 @@ Removes a listener
 Flagger.publish(entity: Entity): void
 ```
 
-Explicitly notify Airdeploy about an Entity
+Explicitly notifies Airdeploy about an Entity
```
Comment (thealveen) [inline docs/api-reference/js.md:93]: All of them

File: docs/api-reference/php.md [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+| sourceURL    | FLAGGER_SOURCE_URL        | https://flags.airdeploy.io/v3/config/       | URL to get `FlaggerConfiguration`                                                                       |
+| backupURL    | FLAGGER_BACKUP_SOURCE_URL | https://backup-api.airshiphq.com/v3/config/ | backup URL to get `FlaggerConfiguration`                                                                |
+| sseURL       | FLAGGER_SSE_URL           | https://sse.airdeploy.io/v3/sse/            | URL for real-time updates of `FlaggerConfiguration` via sse                                             |
+| ingestionURL | FLAGGER_INGESTION_URL     | https://ingestion.airdeploy.io/v3/ingest/   | URL for ingestion                                                                                       |
+| logLevel     | FLAGGER_LOG_LEVEL         | ERROR                                       | set up log level: ERROR, WARN, DEBUG. Debug is the most verbose level and includes all Network requests |
+
+- If not provided default arguments values are used and printed to Debug
+- If second(third …) call of `init` happens:
```
Comment (thealveen) [inline docs/api-reference/php.md:36]: space between second and parantheses

**Review by thealveen — CHANGES_REQUESTED** [review]
minor comments here, code snippets parts look good

File: docs/api-reference/go.md (hunk 1 above)
Comment (jeronimo13) [reply docs/api-reference/go.md:87]: https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/php/#language

"It" or "This function" is implied. If it's incorrect for `Publish` function, are other cases correct where "It' is omitted, like this one:
```
    /**
     * Determines if flag is enabled for entity.
     *
     * @param string $codename
     * @param null $entity
     * @return bool
     */
    public static function isEnabled(string $codename, $entity = null): bool
```

**Pull request merged**